packages/trader/src/Modules/Trading/Components/Elements/Turbos/payout-per-point.jsx

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Money, Text, Popover } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import { connect } from 'Stores/connect';
+
+const PayoutPerPoint = ({ currency, proposal_info, contract_type }) => {
+    const labels = localize('Payout per point');
+    const contract_key = contract_type?.toUpperCase();
+    const stake = proposal_info?.[contract_key]?.number_of_contracts || 0;
+    const message = proposal_info?.[contract_key]?.message || ' ';
+
+    return (
+        <Fieldset className={classNames('payout-per-point')}>
+            <div className='payout-per-point__text-popover'>
+                <Text key={labels} size='xs' weight='normal' className='payout-per-point__text'>
```

Comment:
@hamza-deriv as I can see, we don't map the text, so i suggest that we don't add the key, and i believe we can rename `labels` variable to `label` because it's just one label, right? 🙂
```suggestion
                <Text size='xs' weight='normal' className='payout-per-point__text'>
```